<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SoChat</string>
  </property>
  <widget class="QPushButton" name="m_qt">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>460</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_msg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="membernum">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>130</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Online Member(s) [1]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="m_msgShow">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>551</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="m_anc">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="m_alive">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>150</y>
     <width>201</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_send">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>460</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
